ports/zephyr/boards/xiao_ble_nrf52840_sense.conf

```diff
@@ -0,0 +1,67 @@
+# Kconfig configuration for Seeed Studio's XIAO nRF52840 Sense
+
+CONFIG_NETWORKING=n
+CONFIG_CONSOLE_SUBSYS=n
+
+# for PDM microphone, set these three to y
+# and also enable in overlay
+CONFIG_AUDIO=n
+CONFIG_AUDIO_DMIC=n
+CONFIG_AUDIO_DMIC_NRFX_PDM=n
+
+# Enable Bluetooth
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_DEVICE_NAME="XIAO BLE nRF52840 Sense"
+CONFIG_BT_GATT_DYNAMIC_DB=y
+CONFIG_BT_DEVICE_NAME_DYNAMIC=y
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_L2CAP_TX_MTU=252
+CONFIG_BT_BUF_ACL_RX_SIZE=256
+CONFIG_BT_GATT_ENFORCE_SUBSCRIPTION=n
+
+CONFIG_MICROPY_HEAP_SIZE=98304
+CONFIG_MAIN_STACK_SIZE=8192
+
+# Enable drivers for peripherals
+CONFIG_GPIO=y
+CONFIG_I2C=y
+CONFIG_SPI=y
+CONFIG_PWM=y
+CONFIG_ADC=y
+
+# Filesystem support
+# Enable Zephyr's flash map and littlefs implementation
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_LITTLEFS=y
+
+CONFIG_DISK_ACCESS=n
+
+# Enable MicroPython's VFS and mount a littlefs filesystem on boot
+CONFIG_MICROPY_VFS_LFS2=y
+CONFIG_MICROPY_VFS_FAT=y
```

Comment:
I think these will be enabled by default from `prj.conf`, right?

If so, I suggest removing everything from here that's already the default in `prj.conf`, to keep this file minimal.  Otherwise in the future when people add other boards, they copy this file and there are more and more duplicated settings.